Formula/aircrack-ng.rb

```diff
@@ -2,11 +2,10 @@ class AircrackNg < Formula
   desc "Next-generation aircrack with lots of new features"
   homepage "https://aircrack-ng.org/"
 
-  # We can't update this due to linux-only dependencies in >1.1.
-  # See https://github.com/Homebrew/homebrew/issues/29450
-  url "https://download.aircrack-ng.org/aircrack-ng-1.1.tar.gz"
-  sha256 "b136b549b7d2a2751c21793100075ea43b28de9af4c1969508bb95bcc92224ad"
-  revision 2
+  # If you compile aircrack-ng from source using Gmake, it is compatible with OS X
+  # Issue fixed by compiling aircrack-ng from source
+  url "http://download.aircrack-ng.org/aircrack-ng-1.2-rc4.tar.gz"
```

Comment:
I did, they said:

Hi,

it's rc because it's not final release quality in my opinion. It still contain some bugs I want to fix. Nothing too serious though.

The reason we're using GNU is so that we don't have to write 2 different make files.

Best regards,

Thomas